src/gamestruct/displayable.hpp

```diff
@@ -3,40 +3,53 @@
 #include <ncurses.h>
 
 #include "gamestruct/position.hpp"
-#include "gamestruct/state-watcher.hpp"
+#include "gamestruct/state-watcher.tpp"
```

Comment:
6 | #include "gamestruct/state-watcher.tpp"

errore, state watcher non è pitu in gamestruct

Reply:
@Flecart